docs/5g-on-helium/5g-poc.mdx

```diff
@@ -119,22 +117,155 @@ must:
 speed of at least 10Mbps **AND** a rolling average Latency of 50ms or less.
 
-**A 5G Hotspot that does not meet all three of these requirements will be excluded from rewards
-calculations in the given Reward Period.**
+A 5G Hotspot will perform a Speed Test approximately every 12 hours with the results being averaged
+by the MOBILE PoC Oracle over the previous `6` tests. This moving average is reset following a speed
+test gap of `>12 hours` and a minimum of `2` Speed Test results are required to be eligible for
+Genesis Rewards.
+
+:::note Moving Average
+
+The "Moving Average", also known as a "Rolling Average" is the average of the 6 most recent speed
+test results for each download, upload, and latency.
+
+:::
+
+#### Example Calculation Tables
+
+Here are a set of tables to help visualize all of these components that make up the Speed Test
+result.
+
+:::info
+
+Note that it may take up to 12 hours for newly connected 5G Hotspots to perform a Speed Test and a
+minimum of 2 Speed Tests are required for the MOBILE PoC Oracle to calculate a moving average.
+
+:::
+
+**Successful Speed Test:**
+
+This 5G Hotspot is passing all Speed Test metrics and is providing a high-quality connection.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-success.png')} />
+
+> This 5G Hotspot _will_ be eligible for rewards during all Reward Periods after Test 2.
+
+---
+
+**Missing A Single Speed Test:**
+
+This 5G Hotspot has not reported Speed Test data at Test 5.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-one-miss.png')} />
+
+> This 5G Hotspot _will not_ be eligible for rewards while Test 5 and Test 6 are the most recent.
+
+---
+
+**Missing Multiple Speed Tests:**
+
+This 5G Hotspot has not reported Speed Test data during Test 5, 6, or 7. This is often a result of a
+5G Hotspot being disconnected from the Internet for a long time.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-many-miss.png')} />
+
+> This 5G Hotspot _will not_ be eligible for rewards during the Reward Periods for which the results
+> of Test 5, 6, 7, or 8 are the most recent.
+
+---
+
+**Failing A Speed Test:**
+
+This 5G Hotspot has failed its Speed Test on all metrics in Test 6, but the Moving Average stays
+within the passing range.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-one-fail.png')} />
+
+> This 5G Hotspot _will_ be eligible for rewards during all Reward Periods after Test 2.
+
+---
+
+**Failing Multiple Speed Tests:**
+
+This 5G Hotspot has failed Speed Tests in Test 3, 4, 5, and 6. This has caused the Moving Average to
+fail for Test 6, 7, and 8.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-many-fail.png')} />
+
+> This 5G Hotspot _will not_ be eligible for rewards during the Reward Periods for which the results
+> of Test 6, 7, or 8 are the most recent.
```

Comment:
Based on the table data it won't be eligible for reward before Test 2 as well. Same in a few above examples. 
I think you can say something, in addition to not being eligible for reward due to not enough data on the startup, this Hotspot won't be eligible for rewards due to failed ... and so on.